lunabot_behavior/stuck.py

```diff
@@ -52,7 +52,7 @@ def stuck(self):
             error_publisher.publish(stuck_msg)
 
             # check if trying to go somewhere
-            if self.robot_effort.left_drive >= 15 or self.robot_effort.right_drive >= 15:
+            if self.robot_effort.left_drive >= 15 or self.robot_effort.right_drive >= 15: #TODO RJN - these might need to be higher
```

Comment:
Yes, you're almost certainly correct. That being said, I don't think this file ever gets used. There's a strong argument to be made for deleting it entirely.

Reply:
Updated in fix threshold commit